src/graph.cpp

```diff
@@ -130,7 +134,7 @@ bool Graph::Read( wxInputStream &is )
 	wxDataInputStream ds(is);
 
 	unsigned short identifier = ds.Read16();
-	/*unsigned char ver = */ ds.Read8(); // read the version number, not currently used...
+	unsigned char ver = ds.Read8(); // read the version number, not currently used...
```

Comment:
Please update the comment, as this is now used.

Reply:
updated to version 2